x-pack/plugins/observability/public/pages/cases/index.tsx

```diff
@@ -38,13 +38,13 @@ export const CasesPage = React.memo(() => {
     docsLink: docLinks.links.observability.guide,
   });
 
-  return userPermissions == null || userPermissions?.read ? (
+  return userPermissions.read ? (
```

Comment:
@cnasikas thoughts on this? If the permissions haven't been initialized should we show anything? I suppose this could cause a flicker.

Reply:
I tested and I don't see any flickering even on slow connections. This changes the behavior a bit. Now we show the `CaseFeatureNoPermissions` page if `userPermissions` is `null`. Before we would show the `Cases` page but with `userCanCrud` set to `false`. Basically, `userPermissions = null` was like `read = true` for Cases which is not the case. I think the changes you are proposing make more sense to me.